distribution/branding-plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -9,13 +10,13 @@
  * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
- **/
+**/
 -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
-        >
 
-    <bean id="provider" class="org.codice.ddf.branding.impl.BrandingResourceProviderImpl"/>
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
```

Comment:
It is, in fact. But I know the odds are that someone will copy/paste your blueprint at some point and if they need other blueprint functionality will be missing it. It's less about need for today than future-proofing newbie mistakes.